features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -34,6 +34,7 @@
         <RememberMeTimeout>20160</RememberMeTimeout>
         <PersistanceCleanUpTimeout>20160</PersistanceCleanUpTimeout>
         <PersistanceCleanUpPeriod>1140</PersistanceCleanUpPeriod>
+        <RegistryCleanUpPeriod>1440</RegistryCleanUpPeriod>
```

Comment:
Shall we move this to identity-mgt.properties ? The name here is too generic, it doesn't indicate anything about identity-mgt. May be in next release, once we move all the configurations to UI, the remaining configurations in identity-mgt.properties we can put it to identity.xml and remove that file.